Description: Use absolute path to python2 interpreter for bins
Author: Wikimedia Release Engineering <releng@wikimedia.org>
--- b/bin/compile-wikiversions
+++ a/bin/compile-wikiversions
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Compile local wikiversions.json to wikiversions.php
--- b/bin/deploy
+++ a/bin/deploy
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Deploy service to cluster.
--- b/bin/deploy-local
+++ a/bin/deploy-local
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Deploy service to localhost.
--- b/bin/deploy-log
+++ a/bin/deploy-log
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Tail and filter the deploy.log.
--- b/bin/hhvm-graceful-all
+++ a/bin/hhvm-graceful-all
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Restart HHVM across the entire cluster
--- b/bin/mw-update-l10n
+++ a/bin/mw-update-l10n
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Update localization files.
--- b/bin/mwversionsinuse
+++ a/bin/mwversionsinuse
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Get a list of active MediaWiki versions.
--- b/bin/scap
+++ a/bin/scap
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Sync MW source and config to cluster.
--- b/bin/scap-hhvm-restart
+++ a/bin/scap-hhvm-restart
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Restart HHVM safely
--- b/bin/scap-purge-l10n-cache
+++ a/bin/scap-purge-l10n-cache
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Purge the localization cache for an inactive MediaWiki version
--- b/bin/scap-rebuild-cdbs
+++ a/bin/scap-rebuild-cdbs
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Rebuild localization cache CDB files from the JSON versions.
--- b/bin/scap-say
+++ a/bin/scap-say
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Make the scap pig say stuff.
--- b/bin/security-patch-check
+++ a/bin/security-patch-check
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Tail and filter the deploy.log.
--- b/bin/sync-common
+++ a/bin/sync-common
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Sync MW_COMMON
--- b/bin/sync-dir
+++ a/bin/sync-dir
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Synchronize a directory from common/ to the cluster.
--- b/bin/sync-file
+++ a/bin/sync-file
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Synchronize a file from common/ to the cluster.
--- b/bin/sync-l10n
+++ a/bin/sync-l10n
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Synchronize the localization files for a MediaWiki branch
--- b/bin/sync-master
+++ a/bin/sync-master
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Sync /srv/mediawiki-staging with full git data
--- b/bin/sync-wikiversions
+++ a/bin/sync-wikiversions
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 # -*- coding: utf-8 -*-
 #
 # Rebuild and sync wikiversions.php to the cluster.
